### PR TITLE
build(sass): connotation mixin support

### PR DESCRIPTION
**Pull request by yinonov**
restore connotation sass mixin

### CODE REVIEW

File: src/tokens/build/scss/typography-variables/web.scss [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+*/
+
+$typography-category-list: (
+  'body-1': (
+    'font-family': SpeziaWebVariable,
+    'font-size': 16px,
+    'font-stretch': 50%,
+    'font-weight': 400,
+    'letter-spacing': 0px,
+    'line-height': 24px,
```
Comment (rachelbt) [inline src/tokens/build/scss/typography-variables/web.scss:13]: not sure its relevant to this issue but its better to give a non px value for the line height/ 24px for 16px font size is better as: `line-height:1.5;`

File: src/tokens/build/scss/size/web.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	--vvd-size-4: 32px;
+	--vvd-size-5: 40px;
+	--vvd-size-lg: 48px;
+	--vvd-size-xl: 56px;
+	--vvd-size-2xl: 64px;
+	--vvd-size-3xl: 72px;
+	--vvd-size-md: 40px;
+	--vvd-size-lg: 48px;
+	--vvd-size-xl: 56px;
+	--vvd-size-2xl: 64px;
```
Comment (rachelbt) [inline src/tokens/build/scss/size/web.scss:14]: these two are duplication
what happened to 2xs etc.?

File: src/components/badge/badge.scss [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
+	@include connotation.connotation;
+
+    font:600 ultra-condensed 12px / 16px SpeziaWebVariable;
+    letter-spacing:0px;
+    text-decoration:none;
+    text-transform:none;
+    display:inline-flex;
+    align-items: center;
+    column-gap: 8px;
+    vertical-align: middle;
```
Comment (rachelbt) [inline src/components/badge/badge.scss:18]: not sure this is needed if the badge is flex with items canter

File: src/tokens/build/scss/typography-variables/web.scss (hunk 1 above)
Comment (yinonov) [reply src/tokens/build/scss/typography-variables/web.scss:13]: These are copied tokens. Migrating to rems is on the plate but this is autogenerated by token handler and just hard coded here for the interim

File: src/components/badge/badge.scss (hunk 3 above)
Comment (yinonov) [reply src/components/badge/badge.scss:18]: Apparently it's needed to align itself. That solved the misaligning badges with icons

File: src/tokens/build/scss/size/web.scss (hunk 2 above)
Comment (yinonov) [reply src/tokens/build/scss/size/web.scss:14]: Again, autogenerated tokens copied from vivid legacy. Will change soon...

File: src/components/badge/badge.scss (hunk 3 above)
Comment (rachelbt) [reply src/components/badge/badge.scss:18]: Right! I remember that!

**Pull request merged**